hhs_oauth_server/urls.py

```diff
@@ -48,8 +48,6 @@
         name='warnings'),
 
     url(r'^social-auth/', include('social_django.urls', namespace='social')),
-    # Admin
-    url(r'^admin/', include(admin.site.urls)),
```

Comment:
Why are we removing this?

Reply:
@samgensburg-gov  we havve this line instead  
     

         decorated_url(r'^' + ADMIN_REDIRECTOR + 'admin/', include(admin.site.urls),
                  wrap=staff_member_required(login_url=settings.LOGIN_URL)),

This makes sure our production admin URL is not guessable and is a shared secret.

Reply:
TheADMIN_REDIRECTOR was applied as a result of a finding in the SCA.